### DISCUSSION TITLE
Sub-Servers like Guilded has.

### DISCUSSION THREAD

**Discussion by Larsenticus** [stoatchat/discussions#617] [Feature Suggestions]
If you want to keep your Revolt server organized, get some sub servers too! Because if you get over 1000 members into your server, you can organize it with specific games, roles, etc! Though we know that Revolt isn't meant to be a copycat of all discord, guilded, etc, but it doesn't mean you can't be a better competitor than those!

**Comment by YukitoNakamura** [comment]
That's an interesting feature. I remember seeing it on Guilded.

**Comment by LunaZE** [comment]
god, this would be absolutely AMAZING, honestly guilded was A+++ until stupid roblox decided to murder the platform...

**Reply by YukitoNakamura** [reply]
True, I deleted my account on Guilded after that.

**Reply by bmvoss123** [reply]
+1. Definitely watching this issue.

**Comment by bitrain574** [comment]
Oh please do this! That’ll be helpful!
